website/project/model.py

```diff
@@ -1541,6 +1577,15 @@ def delete_search_entry(self):
             logger.exception(e)
             log_exception()
 
+    def delete_registration_tree(self):
+        self.is_deleted = True
+        if not getattr(self.embargo, 'for_existing_registration', False):
+            self.registered_from = None
+        self.save()
```

Comment:
To me it seems a bit problematic to not save in a recursive function, but for convention's sake I will add the kwarg